xtheadsxstatus.adoc

```diff
@@ -0,0 +1,53 @@
+[#xtheadsxstatus]
+== T-Head extension status register for S-mode (XTheadSxStatus)
+
+[NOTE,caption=Frozen]
+The `XTheadSxStatus` extension is `stable`.
+
+The `XTheadSxStatus` ISA extension provides the `th.sxstatus` CSR that holds
+status information and allows to control T-Head custom extensions.
+
+Extension version: 1.0.
+
+The `th.sxstatus` CSR is 64 bits wide, has the address `0x5C0` and is readable
+and writable in S-mode or lower modes.
```

Comment:
According to C910 User Manual, `th.sxstatus` is a (mostly) read-only shadow of `th.mxstatus`. Only bit 15 (MM), which controls unaligned access, can be changed in S-mode.

Reply:
You are absolutely right, but this is not a contradiction to what is written here.

The RISC-V priv spec defines several custom CSR ranges. One of them is `0x5c0-0x5ff`, which is for custom read/write Supervisor-Level CSRs. And that's what is meant with "is readable and writable in S-mode or lower modes".

Also note, that there is no further specification of "read-only shadow", so writes could trigger an exception, be ignored, etc. The way this is specified in this PR is such that all these behaviors are valid (all bits are defined as WPRI). If we ever have to document the `MM` bit, we can clarify the exact behavior for this bit there.